schutzbot/include_image_info.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euxo pipefail
+sudo dnf install gh -y
+sudo pip install python-gitlab
+
+# Login to GH
+echo "${SCHUTZBOT_GH_TOKEN}" > /tmp/secret.txt
+gh auth login --with-token < /tmp/secret.txt
+
+# import the image info from the current build
+./tools/ci_import --pipeline-id "$CI_PIPELINE_ID" --token  ${GITLAB_TOKEN} --verbose
```

Comment:
Double-space before `${GITLAB_TOKEN}`, I would also quote it just to be extra-safe. (shellcheck would also be nice in this repository)